### DISCUSSION TITLE
I’m facing a database query error while fetching event data

### DISCUSSION THREAD

**Discussion by ajselvan** [hasura/graphql-engine#10637] [Question]
```
query MyQuery {
  events(
    where: {data: {_contains: {user_address: "0x829faddf57a5a6c9f1563e5e3c643f9942039486a9b030517187c7baebd4fc87"}}}
    order_by: {transaction_version: desc}
    limit: 2
  ) {
    transaction_version
    data
    type
  }
}
```

This is my query code to fetch the events. The default limit is 100, right?

When I provide a limit of 100, I get a database query error, even though I only have 2 event entries. If I set the limit to 2, it returns the response correctly.

However, if I set a limit greater than the available event data, it returns an error. The issue is that we don't know the exact length of the event data. Could you please help me resolve this? Thank you!

**Comment by robertjdominguez** [comment]
Closing as this is a dupe of #10636.
